client/starwhale/api/_impl/data_store.py

```diff
@@ -1128,6 +1119,23 @@ def update_table(
             else:
                 table.insert(r)
 
+    def _get_table(self, table_name: str, schema: TableSchema) -> MemoryTable:
+        table = self.tables.get(table_name, None)
+        if table is not None:
+            return table
+        with LocalDataStore._lock:
```

Comment:
the LocalDataStore _lock is the process scope for all of the objects, but self.tables is the object scope,should we replace the class lock with the object lock?

Reply:
done